packages/contracts-bedrock/src/L2/L1Block.sol

```diff
@@ -149,6 +161,9 @@ contract L1Block is ISemver, IGasToken {
             sstore(blobBaseFee.slot, calldataload(68)) // uint256
             sstore(hash.slot, calldataload(100)) // bytes32
             sstore(batcherHash.slot, calldataload(132)) // bytes32
+            sstore(isDeposit.slot, calldataload(133)) // boolean
```

Comment:
We can remove the need to pass the "is deposit" flag in the L1 attributes calldata, that will simplify a bunch